packages/core/examples/inputGroupExample.tsx

```diff
@@ -86,14 +82,16 @@ export class InputGroupExample extends BaseExample<IInputGroupExampleState> {
             <div>
                 <div>
                     <InputGroup
+                        className={className}
                         disabled={disabled}
                         leftIconName="filter"
                         onChange={this.handleFilterChange}
                         placeholder="Filter histogram..."
-                        rightElement={clearButton}
+                        rightElement={filterValue ? <Spinner className={Classes.SMALL} /> : undefined}
                         value={filterValue}
                     />
                     <InputGroup
+                        className={className}
                         disabled={disabled}
                         placeholder="Enter your password..."
```

Comment:
Oh, and just to capture in the same batch of comments: I agree with @leebyp that it'd be sweet to include `text-overflow: ellipsis` etc. on placeholders. That could be a more common issue on large input groups, even though I would hope users would make large input groups commensurately wider to fit their contents.